### PR TITLE
App

### PR DESCRIPTION
**Pull request by KingPhilip14**
Added the documentation for the app module.

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -41,9 +41,10 @@ captions in one place to allow for easier analysis.
    :maxdepth: 2
    :caption: Contents:
 
-   Sources
-   Utils
-   Threads
+   app
+   utils
+   threads
```
Comment (Omarelsegeiny) [inline docs/source/index.rst:46]: We might wanna capitalize these

**Review by Omarelsegeiny — APPROVED** [review]

File: docs/build/html/_sources/app.rst [hunk 2]
```diff
@@ -0,0 +1,109 @@
+App Module
+==========
+
+The `App` Module contains two files in it, `image_processor.py` and `input_manager.py`. When the application starts,
```
Comment (minisodan) [inline docs/build/html/_sources/app.rst:4]: in the initial paragraph you say "its these two files" this is nit picky but id change the wording to not make it seem like you are talking to an audience. maybe something like "these two files initiate..."

File: docs/build/html/_sources/app.rst [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+it's these two files that initiate the process of processing the given images and accepting user input respectively.
+
+
+Input Manager
+-------------
+
+``start()``
+This method will start the application and is called in the `runner.py` file. It will first prompt the user for a
```
Comment (minisodan) [inline docs/build/html/_sources/app.rst:12]: This is a common issue in this file try not to start a documentation with "This method"

File: docs/build/html/_sources/app.rst [hunk 4]
```diff
@@ -0,0 +1,109 @@
+App Module
+==========
+
+The `App` Module contains two files in it, `image_processor.py` and `input_manager.py`. When the application starts,
+it's these two files that initiate the process of processing the given images and accepting user input respectively.
```
Comment (minisodan) [inline docs/build/html/_sources/app.rst:5]: in the initial paragraph you say "its these two files" this is nit picky but id change the wording to not make it seem like you are talking to an audience. maybe something like "these two files initiate..."

**Review by minisodan — CHANGES_REQUESTED** [review]

**Review by minisodan — APPROVED** [review]

**Pull request merged**